packages/api-explorer/lib/generateFakeSchema.js

```diff
@@ -8,6 +8,37 @@ jsf.option({
   alwaysFakeOptionals: true,
 })
 
+function generateObjectExample(example, properties) {
+  const generatedExample = { ...properties ? jsf.generate(properties) : {} }
+  // eslint-disable-next-line guard-for-in
```

Comment:
instead of disabling the lint is there another approach available for this iteration? For instance an Object.entries().forEach?

Reply:
I turned "off" the `guard-for-in` in .eslintrc. If i use other method i call two cycle to get this information.
This linting is not so useful. If i choose to exclude some key from cycle i can do it without his constraint. What do you think?